docker-entrypoint.sh

```diff
@@ -65,6 +65,21 @@ escape_for_replace() {
 }
 
 echo "=="
+
+if [ -f "/var/www/localhost/htdocs/protected/config/dynamic.php" ]; then
+	echo "Generate config using common factory ..."
+
+	echo '<?php return ' \
+		> /var/www/localhost/htdocs/protected/config/common.php
+
+	sh -c "php /var/www/localhost/htdocs/protected/config/common-factory.php" \
+		>> /var/www/localhost/htdocs/protected/config/common.php
+
+	echo ';' \
+		>> /var/www/localhost/htdocs/protected/config/common.php
```

Comment:
Looks good. One last thing: I would use `php -l <filename>` to check the syntax. Some people will manage to break the config and showing this on bootup is better then booting with a broken config (imo).

Reply:
Good idea. Will do.